src/utils/validators.js

```diff
@@ -116,6 +117,13 @@ export const getWalletNameError = (
   }
 }
 
+export const getUnstoppableDomainAddress = async (domain: string): Promise<Address> => {
+  const resolution = new Resolution();
+  try {
+    return await resolution.addr(domain, 'ADA')
```

Comment:
If you support other tokens than ADA it's better to take token name from user input (select?)

Reply:
For Cardano native tokens, would the Cardano 'ADA' address still be valid?
For example, if there was a Cardano native token 'Foo' would you not send 'Foo' to the users 'ADA' address?

If the user selected Cardano native asset 'Foo' would the resolver need to resolve for 'Foo' or would it evaluate for 'ADA'?
Afaik this is true for both Cardano and for Ethereum (which I have the most experience with).

Can you advise on this?

Reply:
We don't support Cardano native tokens yet, so ADA stands for Ethereum version only.

Reply:
Resolved in [297c3b1754a64ce4b81e5739db428adc357a1a18](https://github.com/Emurgo/yoroi-mobile/pull/1465/commits/297c3b1754a64ce4b81e5739db428adc357a1a18)

Reply:
> For Cardano native tokens, would the Cardano 'ADA' address still be valid?
> For example, if there was a Cardano native token 'Foo' would you not send 'Foo' to the users 'ADA' address?

Yes.

> If the user selected Cardano native asset 'Foo' would the resolver need to resolve for 'Foo' or would it evaluate for 'ADA'?

I think it makes sense for now to just evaluate everything for `ADA`, as it is more reasonable to expect users would have general Cardano addresses registered under `ADA` for anything including native tokens. If ever there's a case when users would want the same domain name resolve to a different Cardano address depending on the selected asset - we can address it then if deemed reasonable.